locustfile.py

```diff
@@ -200,6 +200,20 @@ def get_mur(self, term=None):
         }
         self.client.get('legal/docs/murs/7074', name='legal_get_mur', params=params)
 
+    @locust.task
+    def get_adr(self, term=None):
+        params = {
+            'api_key': API_KEY,
+        }
+        self.client.get('legal/docs/adrs/668', name='legal_get_adr', params=params)
```

Comment:
Again pre-existing - we may want to select a random document from a list of known document numbers. Also, applies to AOs, AFs and MURs.

Reply:
I agree, I'll put in a new commit.